frontend/src/metabase/search/components/InfoText.jsx

```diff
@@ -46,6 +46,8 @@ export function InfoText({ result }) {
       return jt`Segment of ${(<TableLink result={result} />)}`;
     case "metric":
       return jt`Metric for ${(<TableLink result={result} />)}`;
+    case "action":
+      return jt`for ${result.model_name}`;
```

Comment:
I assume this should involve a helper function, but I decided I'd leave it for a real frontender :)